### PR TITLE
move model.go into model package

### PR DESCRIPTION
**Pull request by RipulHandoo**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/experimental/experimental.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ package experimental
 import (
 	"fmt"
 
-	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/components"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/models"
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/experimental/experimental.go:20]: If you are removing components, then why you're referencing that in Line no 48? Explain

File: mesheryctl/internal/cli/root/models/view.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
+package model
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/models/view.go:1]: Make sure if you're naming a package, it must match the name of directory too. If it's `model` please rename the directory to `model` and not `models`

**Review by alphaX86 — CHANGES_REQUESTED** [review]

File: mesheryctl/internal/cli/root/experimental/experimental.go (hunk 1 above)
Comment (RipulHandoo) [reply mesheryctl/internal/cli/root/experimental/experimental.go:20]: It must have been a mistake on my part. Corrected.

File: mesheryctl/internal/cli/root/models/view.go (hunk 2 above)
Comment (RipulHandoo) [reply mesheryctl/internal/cli/root/models/view.go:1]: Noted.

File: mesheryctl/internal/cli/root/experimental/experimental.go [hunk 3]
```diff
@@ -45,6 +46,6 @@ var ExpCmd = &cobra.Command{
 }
 
 func init() {
-	availableSubcommands = append(availableSubcommands, system.ModelCmd, components.ComponentsCmd)
+	availableSubcommands = append(availableSubcommands, system.ModelCmd, components.ComponentsCmd, model.ModelCmd)
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/experimental/experimental.go:49]: Remove models from system

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
There might be a couple more lint check items here...

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
@RipulHandoo Mesheryctl build is failing. Did you tested these changes?

File: mesheryctl/internal/cli/root/models/export.go [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+		modelReader := strings.NewReader(strcontent)
+		ymlModel, err := io.ReadAll(modelReader)
+		if err != nil {
+			utils.Log.Error(handlers.ErrIOReader(err))
+			return err
+		}
+
+		if _, err := file.Write(ymlModel); err != nil {
+			utils.Log.Error(handlers.ErrWritingIntoFile(err, tmpModelFile))
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/models/export.go:148]: @RipulHandoo Return the same error?

File: mesheryctl/internal/cli/root/models/export.go [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+			utils.Log.Error(handlers.ErrIOReader(err))
+			return err
+		}
+
+		err = os.WriteFile(outputPath+"/"+pretifiedName, content, 0644)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to write oci image data to file: %v\n", err)
+			os.Exit(1)
+		}
+		fmt.Printf("OCI Image %s is successfully built on %s \nDigest: %v, Size: %v", pretifiedName, outputPath, digest, size)
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/models/export.go:200]: Can we move away from fmt?

File: mesheryctl/internal/cli/root/models/view.go [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+		outFormatFlag = strings.ToLower(outFormatFlag)
+		if outFormatFlag == "yaml" {
+			if output, err = yaml.Marshal(selectedModel); err != nil {
+				return errors.Wrap(err, "failed to format output in YAML")
+			}
+			fmt.Print(string(output))
+		} else if outFormatFlag == "json" {
+			return outputJson(selectedModel)
+		} else {
+			return errors.New("output-format choice invalid, use [json|yaml]")
```
Comment (MUzairS15) [inline mesheryctl/internal/cli/root/models/view.go:125]: This isn't meshkit error.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by MUzairS15** [comment]
@RipulHandoo Reviews incorporated?

**Pull request closed without merging**